src/arkouda_server.chpl

```diff
@@ -124,6 +123,48 @@ proc main() {
         stdout.flush();
     }
 
+    /**
+     * Register our server commands in the CommandMap
+     * There are 4 general types
+     * 1. Standard, required commands which adhere to the standard Message signature
+     * 2. Specialized, required commands which do not adhere to the standard Message signature
+     * 3. "Backbone"/default modules we have deemed should be included in every build
+     * 4. "Optional" modules which are included at compilation time via ServerModules.cfg
+     */
+    proc registerServerCommands() {
+        registerBinaryFunction("tondarray", tondarrayMsg);
+        registerFunction("create", createMsg);
+        registerFunction("delete", deleteMsg);
+        registerFunction("arange", arangeMsg);
+        registerFunction("linspace", linspaceMsg);
+        registerFunction("set", setMsg);
+        registerFunction("info", infoMsg);
+        registerFunction("str", strMsg);
+        registerFunction("repr", reprMsg);
+        registerFunction("getconfig", getconfigMsg);
+        registerFunction("getmemused", getmemusedMsg);
+        registerFunction("getCmdMap", getCommandMapMsg);
+        registerFunction("clear", clearMsg);
+
+        // For a few specialized cmds we're going to add dummy functions,
+        // they will be intercepted in the cmd processing select statement
+        registerFunction("array", akMsgSign);
```

Comment:
Are they only being registered in the map here so they can be displayed when the user requests to print out all the commands?

Reply:
Yes.  It's a little clunky, but the alternative is to hard-code some stuff in CommandMap which I didn't want to do.